tests/connect/standalone-x509-auth-001.phpt

```diff
@@ -0,0 +1,33 @@
+--TEST--
+Connect to MongoDB with SSL and X509 auth
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; NEEDS("STANDALONE_X509"); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$SSL_DIR = realpath(__DIR__ . '/../../scripts/ssl/');
+
+$driverOptions = [
+    // libmongoc does not allow the hostname to be overridden as "server"
+    'allow_invalid_hostname' => true,
+    'weak_cert_validation' => false,
+    'ca_file' => $SSL_DIR . '/ca.pem',
+    'pem_file' => $SSL_DIR . '/client.pem',
+    // TODO: this doesn't appear to have any effect. Does the PEM file not have a password?
+    'pem_pwd' => 'qwerty',
```

Comment:
Forgot to remove this before merging. Will be addressed in https://github.com/mongodb/mongo-php-driver/pull/376.